tests/libnet/plumbing.go

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package libnet
+
+import (
+	"context"
+	"fmt"
+
+	nadv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+func CreateNAD(virtClient kubecli.KubevirtClient, namespace, nadName string) error {
+	nadSpec := newNetworkAttachmentDefinition(nadName)
+	_, err := virtClient.NetworkClient().K8sCniCncfIoV1().NetworkAttachmentDefinitions(namespace).Create(
+		context.TODO(),
+		nadSpec,
+		metav1.CreateOptions{},
+	)
+	return err
+}
+
+func newNetworkAttachmentDefinition(networkName string) *nadv1.NetworkAttachmentDefinition {
+	config := fmt.Sprintf(`{"cniVersion": "0.3.1", "name": %q, "type": "cnv-bridge", "bridge": %q}`, networkName, networkName)
```

Comment:
This is hiding the fact that it uses a specific CNI (`cni-bridge`).

As you placed it in a library that others may use, please make it as general as possible (e.g. receiving these details from the caller).

If you prefer not to do this, then this one fits the specific test that called it, not this test network package.